torch/_inductor/quantized_lowerings.py

```diff
@@ -24,10 +41,59 @@ def register_quantized_ops():
 
 
 def register_woq_mm_ops():
-    lowering.add_needs_realized_inputs(
-        [
-            aten._weight_int8pack_mm,
-        ]
-    )
+    @register_lowering(aten._weight_int8pack_mm, type_promotion_kind=None)
+    def int8pack_mm(input, weight, scale, *, layout=None):
+        _, _, _, layout, mat1, mat2, scale = mm_args(
+            input, weight, scale, layout=layout, trans_w=True
```

Comment:
> Is the weight also as size `[*b2, n, k2]` instead of `*b2, k2, n` with TorchAO?
Yes


> Why we need to pass `scale` into `mm_args`?

Will remove, thanks!